platform/ui-next/src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -88,7 +88,12 @@ const Thumbnail = ({
                   loadingProgress && loadingProgress < 1 && 'bg-primary/25'
                 )}
               ></div>
-              <div className="text-[11px] font-semibold text-white">{modality}</div>
+              <div
+                className="text-[11px] font-semibold text-white"
+                data-cy="series-modality-label"
```

Comment:
If this is for a particular series consider adding something to distinguish it from the other series thumbnail, like series instance uid or something. If the locator though is straightforward, then maybe this is not so big a deal, but something to consider. This comment applies to the other data-cy values in this file too.

Reply:
Not actually, the idea is to be generic, and them from the selectors I can filter the one I want based on the Modality or Series Name, this is the pattern I'm more used to see on Cypress/Selenium tests at least.